schemes/rasch_kristjansson/rk_stratiform.F90

```diff
@@ -0,0 +1,571 @@
+! Copyright (C) 2025 University Corporation for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+!
+! Rasch and Kristjansson prognostic cloud microphysics and CAM4 macrophysics
+! CCPP-ized: Haipeng Lin, January 2025
+module rk_stratiform
+  use ccpp_kinds, only: kind_phys
+
+  implicit none
+  private
+  save
+
+  ! public CCPP-compliant subroutines
+  !   note: cloud_fraction_perturbation_run is NOT a fully CCPP-compliant scheme
+  !         because it calls the compute_cloud_fraction scheme for perturbation.
+  !         this is currently not possible to elegantly handle in the framework.
```

Comment:
Can you elaborate on this?  My understanding is that listing `compute_cloud_fraction` as a dependency in the metadata file is a valid CCPP action, and should help enforce the fact that `compute_cloud_fraction` is required for `cloud_fraction_perturbation_run` even if not specifically listed in the SDF.  Is my understanding incorrect here, or am I just not understanding the core issue?

Reply:
Is there a decision somewhere that a CCPP-compliant scheme cannot also be a utility (i.e.,subroutine registered in the metadata as a dependency)?

Reply:
I think for now I would vote to just remove the "not CCPP compliant" comment, as technically the scheme will still be portable even with the dependency (which is usually my definition of whether something is "compliant" or not).  We can certainly revisit this in the future if we think the way it is currently organized is not ideal.  Thanks!